pennylane/optimize/qng.py

```diff
@@ -86,6 +87,53 @@ class QNGOptimizer(GradientDescentOptimizer):
         * For multi-QNode models, we don't know what geometry is appropriate
           if a parameter is shared amongst several QNodes.
 
+        If the objective function is VQE/VQE-like, i.e. a function of a group
+        of QNodes that share an ansatz, there are two ways to use the optimizer:
+
+        * Realize the objective function as a VQECost object, which has
+          the metric_tensor function.
+
+        * Manually provide the metric_tensor_fn as the metric_tensor of
+          one of the QNodes in the group of QNodes involved in the objective function.
```

Comment:
Maybe it makes sense to make this a bit vaguer? Something like the following? Although, it places the burden on the user to make sure that they are passing a metric tensor function that is physically valid!
```suggestion
        * Manually provide the ``metric_tensor_fn`` corresponding to the metric tensor of
          of the QNode(s) involved in the objective function.
```